packages/next/client/image.tsx

```diff
@@ -438,6 +438,27 @@ export default function Image({
           ) : null}
         </div>
       ) : null}
+      {!isVisible && (
+        <noscript>
+          <img
+            {...rest}
+            {...generateImgAttrs({
+              src,
+              unoptimized,
+              layout,
+              width: widthInt,
+              quality: qualityInt,
+              sizes,
+              loader,
+            })}
+            src={src}
+            decoding="async"
+            sizes={sizes}
+            style={{ ...imgStyle, visibility: 'inherit' }}
```

Comment:
I think we can remove visibility since it was removed in https://github.com/vercel/next.js/pull/23278

cc @shuding

Reply:
Yep!